matrixUtils/index.js

```diff
@@ -0,0 +1,80 @@
+//==========================================================================
+// MATRIX UTIlS PACKAGE
+//==========================================================================
+// author: Sam Bernal
+// version: 1.0
+// description: methods to help creation and loop through multidimensional arrays
+//==========================================================================
+
+//calling it matrix since it's for only multidimensional arrays
+//choosing old ES5 function call for scoping
+//has 3 parameters
+//size for X(row)
+//size for Y(column)
+//value, a function that will determine the value of each elemenet
+module.exports = {
+	//==========================================================================
```

Comment:
I suggest you look into JS Doc format for describing your code: it's more standard/portable, there are lots of convenience packages that work with it.

https://jsdoc.app/about-getting-started.html